Loading/ru.lproj/Localizable.strings

```diff
@@ -1 +1 @@
-"APPLICATION_NOT_RESPONDING" = "Программа не отвечает";
"PREFS" = "Настройки...";
"DISPLAY_AS" = "Отобразить";
"HIDE" = "Скрыть";
"HIDE_NAME" = "Скрыть «%@»";
"HIDE_OTHERS" = "Скрыть остальные";
"OPEN_AT_LOGIN" = "Открывать при входе";
"OPTIONS" = "Параметры";
"QUIT" = "Завершить";
"SHOW_IN_FINDER" = "Показать в Finder";
"SORT_BY" = "Сортировать по";
"TURN_NOTIFICATIONS_OFF" = "Выключить уведомления";
"TURN_NOTIFICATIONS_ON" = "Включить уведомления";
"WELCOME_TO_LOADING" = "Добро пожаловать в Loading!";
"LOADING_INTRO" = "Loading показывает, когда приложения используют Интернет. Вы хотите, чтобы открыть, когда вы войти ли?";

"CANCEL" = "Отменить";
"LOADING" = "Загрузка";
"LOADED" = "загружен";

"APPLICATIONS" = "Программы";
"SYSTEM" = "Система";

"CHECK_FOR_UPDATES" = "Проверять наличие обновлений";
"SEND_FEEDBACK" = "Оставить отзыв...";
"ABOUT" = "Об %@";
"THANKS" = "Спасибо!";

"MORE_INFO" = "Подробнее...";
"DOWNLOAD" = "Загрузить";
"NAME" = "Имя";
\ No newline at end of file
+"APPLICATION_NOT_RESPONDING" = "Программа не отвечает";
"PREFS" = "Настройки…";
"DISPLAY_AS" = "Показать как";
"HIDE" = "Скрыть";
"HIDE_NAME" = "Скрыть «%@»";
"HIDE_OTHERS" = "Скрыть остальные";
"OPEN_AT_LOGIN" = "Открывать при входе";
"OPTIONS" = "Параметры";
"QUIT" = "Завершить";
"SHOW_IN_FINDER" = "Показать в Finder";
"SORT_BY" = "Сортировать по";
"TURN_NOTIFICATIONS_OFF" = "Выключить уведомления";
"TURN_NOTIFICATIONS_ON" = "Включить уведомления";
"WELCOME_TO_LOADING" = "Добро пожаловать в Loading!";
"LOADING_INTRO" = "Loading показывает, когда ваши приложения используют Интернет.\nХотите, чтобы он запускался при старте системы?";

"CANCEL" = "Отменено";
"LOADING" = "Загружается";
"LOADED" = "Загружено";

"APPLICATIONS" = "Программы";
"SYSTEM" = "Система";

"CHECK_FOR_UPDATES" = "Проверять наличие обновлений";
"SEND_FEEDBACK" = "Оставить отзыв…";
"ABOUT" = "О %@";
"THANKS" = "Спасибо!";

"MORE_INFO" = "Подробнее…";
"DOWNLOAD" = "Загрузить";
"NAME" = "Имя";
```

Comment:
Hm, I need to remember to fix the line endings on these files so the diffs will be more readable. The translations look solid, though!